src/main/java/util/CommunicateWithUser.java

```diff
@@ -0,0 +1,96 @@
+package util;
+
+import DataModel.DataFromForexApi;
+import DataModel.ExchangeRate;
+import Mappers.JSONMapper;
+import services.FetchToTheForexAPI;
+import services.SaveToCsvFile;
+
+import java.util.Scanner;
+
+public class CommunicateWithUser {
```

Comment:
Naming of the class is more suited to a method/verb. It would be better to use a noun, like 'UserCommunicationUtil'

Reply:
Done